app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1> Mechanic Info </h1>
+
+<p>Name: <%= @mechanic.name %></p>
+<p>Years of Experience: <%= @mechanic.years_experience %></p>
+
+<p>Assigned Rides</p>
+  <% @mechanic.current_rides.each do |ride| %>
+    <p><%= ride.name %></p>
+<% end %>
+
+<h2> Add Ride To Workload </h2>
+
+<%= form_with url: mechanic_path(@mechanic), method: :patch, local: true do %>
+    <%= label_tag :add_ride, "Add Ride" %>
+    <%= text_field_tag :add_ride %>
+    <%= submit_tag "Submit" %>
```

Comment:
This form looks good, except for the `_tag`s here. We want to use the form syntax, `form.label :ride_id`. Review the lesson on forms to see the proper syntax.

Reply:
This will work, but it's not the proper syntax.